index.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width" />
+    <title>Setup and Mobile version skeleton</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <section class="header">
+      <a class="name" href="assets/images/Vahan.jpg"
+        ><p>Vahan Khachvankian</p></a
+      >
+      <nav class="burger">
+        <a href="#"><img src="assets/images/lines-56763.png" alt="logo" /></a>
+      </nav>
+    </section>
```

Comment:
- [ ] Please the section should be wrapped in a `header` tag, not a `section ` tag🙏 sorry for the confusion caused here👍

Reply:
Hi, Aziz, 
Thank you for your comments, but would you like to dispel some of my doubts. The previous reviewer said to replace this tag with a section tag, You say that there should be a header, I agree with you, but how can I be sure that the next reviewer will not say that there should be another one?

Reply:
Hello @Gegardus,
I totally understand your concern here. According to the project requirement the first section is called the `header` or `toolbar` section it is only right to wrap that in a `header` tag to make your code more readable, understandable and semantic✌

Reply:
Thank you. I got it. Our conversation will be an alibi for me (just a joke) 
![image](https://user-images.githubusercontent.com/52704856/144006038-cd3367fb-1b46-4782-ab80-c62b4c8f6789.png)

Reply:
Changed.